src/System.Security.Cryptography.Algorithms/tests/AesCcmTests.cs

```diff
@@ -0,0 +1,602 @@
+                Tag = "2dd6ef1c45d4ccb723dc074414db506d".HexToByteArray(),
+            },
+            new AEADTest
+            {
+                Source = $"{NistCcmTestVectors} - VNT192.rsp",
+                CaseId = 50,
+                Key = "d49b255aed8be1c02eb6d8ae2bac6dcd7901f1f61df3bbf5".HexToByteArray(),
+                Nonce = "1af29e721c98e81fb6286370".HexToByteArray(),
+                Plaintext = "062eafb0cd09d26e65108c0f56fcc7a305f31c34e0f3a24c".HexToByteArray(),
+                AssociatedData = "64f8a0eee5487a4958a489ed35f1327e2096542c1bdb2134fb942ca91804c274".HexToByteArray(),
+                Ciphertext = "721344e2fd05d2ee50713531052d75e4071103ab0436f65f".HexToByteArray(),
+                Tag = "0af2a663da51bac626c9f4128ba5ec0b".HexToByteArray(),
+            },
+            new AEADTest
+            {
+                Source = $"{NistCcmTestVectors} - VNT192.rsp",
+                CaseId = 51,
+                Key = "d49b255aed8be1c02eb6d8ae2bac6dcd7901f1f61df3bbf5".HexToByteArray(),
+                Nonce = "ca650ed993c4010c1b0bd1f2".HexToByteArray(),
+                Plaintext = "fc375d984fa13af4a5a7516f3434365cd9473cd316e8964c".HexToByteArray(),
+                AssociatedData = "4efbd225553b541c3f53cabe8a1ac03845b0e846c8616b3ea2cc7d50d344340c".HexToByteArray(),
+                Ciphertext = "5b300c718d5a64f537f6cbb4d212d0f903b547ab4b21af56".HexToByteArray(),
+                Tag = "ef7662525021c5777c2d74ea239a4c44".HexToByteArray(),
+            },
+            new AEADTest
+            {
+                Source = $"{NistCcmTestVectors} - VNT192.rsp",
+                CaseId = 52,
+                Key = "d49b255aed8be1c02eb6d8ae2bac6dcd7901f1f61df3bbf5".HexToByteArray(),
+                Nonce = "318adeb8d8df47878ca59117".HexToByteArray(),
+                Plaintext = "610a52216f47a544ec562117e0741e5f8b2e02bc9bc9122e".HexToByteArray(),
+                AssociatedData = "feccf08d8c3a9be9a2c0f93f888e486b0076e2e9e2fd068c04b2db735cbeb23a".HexToByteArray(),
+                Ciphertext = "83f14f6ba09a6e6b50f0d94d7d79376561f891f9a6162d0f".HexToByteArray(),
+                Tag = "8925c37cc35c1c8530b0be4817814a8e".HexToByteArray(),
+            },
+            new AEADTest
+            {
+                Source = $"{NistCcmTestVectors} - VNT192.rsp",
+                CaseId = 60,
+                Key = "36ad1e3fb630d1b1fbccfd685f44edd8984427b78deae7a9".HexToByteArray(),
+                Nonce = "3af625df8be9d7685a842f260e".HexToByteArray(),
+                Plaintext = "8b9db1c8f9b4892a5654c85467bcffa2e15e28392c938952".HexToByteArray(),
+                AssociatedData = "308443033ecd4a814475672b814b7c6d813d0ec2a0caeecbcaba18a2840cdb6c".HexToByteArray(),
+                Ciphertext = "6bc6890fee299c712fb8d9df9c141f24ee1572b8f15112c2".HexToByteArray(),
+                Tag = "f8c99ccf2d82788cf613a61d60dae458".HexToByteArray(),
+            },
+            new AEADTest
```

Comment:
I don't think there were any 256-bit ones with no plaintext, and no 256-bit ones with no associated data.  (NIST if available, manual if not)

I don't remember if I saw any tests for no AD with no plaintext.

Reply:
DVPT256.rsp have them (Case 0 and 60)